internal/plugin/plugin.go

```diff
@@ -79,6 +79,50 @@ func (cp *CaptivePortal) Apply(ra *ndp.RouterAdvertisement) error {
 	return nil
 }
 
+type PREF64 struct {
+	Inner *ndp.PREF64
+}
+
+func NewPREF64(prefix netip.Prefix, maxInterval time.Duration) *PREF64 {
```

Comment:
Mind adding a table-driven unit test for this function specifically? I want to exercise the various maxInterval calculation paths. Thanks!

Reply:
Added!